src/sdk/pynni/nni/compression/torch/speedup/infer_shape.py

```diff
@@ -612,12 +826,15 @@ def conv2d_mask(module_masks, mask):
     CoarseMask, CoarseMask
         The mask of its input tensor, the mask of its output tensor
     """
-    def convert_to_coarse_mask(mask):
+    def convert_to_coarse_mask(mask, dim=0):
```

Comment:
Maybe we can generalize this function in the future?
`convert_to_coarse_mask` is a useful utility, given a mask, it should be able to convert this mask to an coarse_mask automatically without the parameter `dim`.